soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3251,7 +3251,10 @@ void GenerateRandomizerImgui() {
     cvarSettings[RSK_CUCCO_COUNT] = CVar_GetS32("gRandomizeCuccosToReturn", 7);
     cvarSettings[RSK_BIG_POE_COUNT] = CVar_GetS32("gRandomizeBigPoeTargetCount", 10);
 
-    cvarSettings[RSK_SKIP_CHILD_STEALTH] = CVar_GetS32("gRandomizeSkipChildStealth", 0);
+    // If we skip child zelda, skip child stealth is pointless, so this needs to be reflected in the spoiler log
+    cvarSettings[RSK_SKIP_CHILD_STEALTH] =
+        ((CVar_GetS32("gRandomizeSkipChildZelda", 0) == 0) && CVar_GetS32("gRandomizeSkipChildStealth", 0));
```

Comment:
can we make this just
`!(CVar_GetS32("gRandomizeSkipChildZelda", 0)` instead? i almost missed the `== 0` and thought the logic was wrong

Reply:
I copied it from the weird egg one. Should I change that one too?

Reply:
don't change that one in this PR, if we want to do some general cleanup later we can